### DISCUSSION TITLE
eos/bos index confusion

### DISCUSSION THREAD

**Discussion by olvb** [speechbrain/speechbrain#1698] [Q&A]
Hi everybody,

I see many recipes (for instance the CommonVoice seq2seq recipes) where we use specific values for eos/bos indices (say 1 and 2), but the SentencePiece tokenizer isn't passed these specific values when trained. 
Even in the ASR from scratch colab tutorial, the SentencePiece logs show `bos_id: -1` and  `eos_id: -1` but then in the rest of the tutorial we use `bos_index: 0` and `eos_index: 0`.

Is that a bug or am I missing something?

EDIT: ok according to the docs leaving eos/bos to -1 should be the same as setting it to 0 but I still wonder about the recipes assuming eos/bos equal to 1/2 when that's not the case

**Comment by Adel-Moumen** [comment]
@TParcollet
